_episodes/01-intro.md

```diff
@@ -54,9 +54,9 @@ This means that, for example:
 ## Types of data
 Python knows various types of data. Three common ones are:
 
-* integer numbers
-* floating point numbers, and
-* strings.
+* integer numbers (`int`)
+* floating point numbers (`float`), and
+* strings (`str`).
```

Comment:
I like this addition of the Python names for the types.

Reply:
Yes. Actually it is a very good idea. It is often not very obvious for novices to find out that string is `str`.

Reply:
I don't mind introducing `int`, `float`, and `str`, but we should provide some narrative that instructors could use when talking about them -- putting them in parentheses leaves this explanation up to an individual instructor. I think that it'll be easier to talk about them right after this list ... where we could also introduce the `type` function but this might be out of scope of this PR.